tools/ddsrouter_tool/test/compose/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Add README to this test path to understand the structure (explain which bash/docker/python executes which and what it validates)

Reply:
Pending...